01 OperatorOverloading/OperatorOverloading.Host/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OperatorOverloading.Model;
+namespace OperatorOverloading.Host
+{
+    class Program
+    {
+        public static void Main(string[] args)
+        {
+
+            double amountTemporary;
+            try
+            {
+                Money moneyObject = new Money();
+
+                //Take input First Amount
+                Console.WriteLine("Enter Amount");
+                if (Double.TryParse(Console.ReadLine(), out amountTemporary))
+                    moneyObject.Amount = amountTemporary;
+
+                moneyObject.Currency = (Console.ReadLine());      
```

Comment:
Add null and empty checks for Currency.
Also check string length should be 3 char